includes/page_top.php

```diff
@@ -3,6 +3,8 @@
     $page_request = $_GET['page_request'] ?? \FirstAide\Router::INDEX;
     $query = $_GET['query'] ?? '';
     $page = FirstAide\Router::getPage($UserObj, $page_request, $query);
+    
+    $page['javascripts'][] = 'error.js';
 
```

Comment:
The $_page['title] is being set by in Router.php for every condition and for each page. But in other cases, the default title is the name of the website i.e FirstAide, Peacecorps. You can check it, it is never blank. @npari